carbon-design-system/carbon
# feat(cli): add component command

Reference #7360 

This PR creates an initial command for creating a component. This does not include the package scaffold, instead it scaffolds out files inside of our react project following our current practices.

#### Changelog

**New**

- Add `component` command to `@carbon/cli`

**Changed**

**Removed**

#### Testing / Reviewing

- Try and create your own component with it! You can reference the CLI with the path `./packages/cli/bin/carbon-cli.js` and can call it directly from your shell
- See if tests and stories work 😬
